specification/azurestackhci/AzureStackHCI.StackHCIVM.Management/models.tsp

```diff
@@ -1193,6 +1274,21 @@ model VirtualMachineInstancePropertiesHardwareProfileDynamicMemoryConfig {
   targetMemoryBuffer?: int32;
 }
 
+#suppress "@azure-tools/typespec-azure-core/casing-style" "GPU is a short form. It is the correct case."
+@added(Versions.v2024_05_01_preview)
+@doc("GPU properties - describes the GPU configuration.")
+model VirtualMachineInstancePropertiesHardwareProfileVirtualMachineGPU {
+  @doc("GPU assignment type")
+  assignmentType: GpuAssignmentTypeEnum;
+
+  #suppress "@azure-tools/typespec-azure-core/casing-style" "MB is a short form."
+  @doc("Size of gpu partition in MB for GPU-P")
+  partitionSizeMB?: int64;
+
+  @doc("Name of the GPU")
+  name?: string;
```

Comment:
>name [](http://example.com/codeflow?start=2&length=4)

[ARMBlockingComment] Don't reuse top-level ARM property names within the property bag. This needs to be named something other than "name", e.g. gpuName.

Reply:
If we make this change to rename to gpuName, we would have to update RPaaSMaster branch with a separate ARM review and rollout as well. We are tight on timeline to release and taking this change again to RPaaSMaster will cost us to miss the release. Is this a blocking and must have change?

Reply:
https://github.com/Azure/azure-rest-api-specs-pr/pull/19277/files We've opened this PR to RPSaaSMaster if this is blocking. We kindly request an expedited review of this PR given this was not caught during the earlier review to RPSaaSMaster.